basic/src/test/java/org/jboss/hal/testsuite/test/runtime/deployments/DomainManagedDeploymentsTestCase.java

```diff
@@ -97,151 +138,105 @@ public void createDeployment() throws Exception {
     }
 
     @Test
-    @InSequence(1)
     public void assignDeploymentToServerGroup() throws Exception {
-        navigation.resetNavigation();
-        navigation.step(FinderNames.BROWSE_BY, "Server Groups")
-                .step(FinderNames.SERVER_GROUP, "main-server-group")
-                .step(FinderNames.DEPLOYMENT);
-        navigation.selectColumn().invoke("Add");
+        page.navigateToDeploymentColumnInServerGroup(MAIN_SERVER_GROUP).invoke("Add");
 
         DeploymentWizard wizard = Console.withBrowser(browser).openedWizard(DeploymentWizard.class);
 
         boolean result = wizard.switchToRepository()
                 .nextFluent()
+                .enableDeploymentInGroup(DEPLOYMENT_ASSIGN_NAME)
                 .finish();
 
         assertTrue("Deployment wizard should close", result);
-        ops.verifyIsDeploymentAssignedToServerGroup(MAIN_SERVER_GROUP, NAME);
+        ops.verifyIsDeploymentAssignedToServerGroup(MAIN_SERVER_GROUP, DEPLOYMENT_ASSIGN_NAME);
     }
 
     @Test
-    @InSequence(2)
     public void enableDeployment() throws Exception {
-        navigation.resetNavigation();
-        navigation.step(FinderNames.BROWSE_BY, "Server Groups")
-                .step(FinderNames.SERVER_GROUP, "main-server-group")
-                .step(FinderNames.DEPLOYMENT, NAME);
-        navigation.selectRow().invoke("Enable");
+        //disable deployment first in server group, so it can be enabled in test
+        operations.invoke("undeploy", Address.of("server-group", MAIN_SERVER_GROUP)
+                .and("deployment", DEPLOYMENT_DISABLED_NAME))
+                .assertSuccess();
+
```

Comment:
You are claiming that you are only disabling deployment. But you are actually undeploying it (just leaving content on server). The comment should match to what is being really done.